README.md

```diff
@@ -58,11 +58,37 @@ coding style used in typeshed.
 
 This contains stubs for modules in the Python standard library -- which
 includes pure Python modules, dynamically loaded extension modules,
-hard-linked extension modules, and the builtins. `VERSIONS` file lists
-the oldest *supported* Python version where the module is available.
-In the `stdlib/@python2` subdirectory you can find Python 2 versions of
-the stub files that must be kept different for Python 2 and 3, like
-`builtins.pyi`.
+hard-linked extension modules, and the builtins. The `VERSIONS` file lists
+the versions of Python where the module is available.
+
+The structure of the `VERSIONS` file is as follows:
+- Blank lines and lines starting with `#` are ignored.
```

Comment:
Do we really need this? Currently there are no comments or blank lines, although most type checker seem to support this.

Reply:
I feel like it's safer for any human-readable file format to support comments, since at some point you're going to want them.